src/design-system/color/palettes.ts

```diff
@@ -474,7 +473,6 @@ export const foregroundColors: Record<
   'yellow': selectBackgroundAsForeground('yellow'),
   'fill': selectBackgroundAsForeground('fill'),
   'fillSecondary': selectBackgroundAsForeground('fillSecondary'),
-  'fillTertiary': selectBackgroundAsForeground('fillTertiary'),
```

Comment:
Even though `fillTertiary` was renamed to `surfaceSecondaryElevated`, it was removed here rather than being renamed since all surface colors are only ever intended to be used as background colors, not foreground.